pyMBE.py

```diff
@@ -3213,6 +3214,10 @@ def setup_df (self):
                 '': object},
             'l0': {
                 '': float},
+            'node_map':{
+                '':object},
+            'chain_map':{
+                '':object}
```

Comment:
@1234somesh I believe this lines have leaked from your work in the hydrogel feature. Could you please remove them from this branch and add them in the branch where you work in the feature? Otherwise the unit test `testsuite/read-write-df_test.py` crashes because one needs to adapt it to the new columns in `pmb.df`.